bin/get-scitokens-mapfile

```diff
@@ -43,12 +49,15 @@ def main(argv):
         for token_issuer in vo_data["Credentials"]["TokenIssuers"]:
             url = token_issuer.get("URL")
             if url:
-                url = f'"{url}"'
+                if args.regex:
+                    url = f'/{re.escape(url)}/'
```

Comment:
```suggestion
                    url = f'/^{re.escape(url)},/'
```

@edquist and I verified that the regex acts more like `re.search` instead of `re.find` so we should:

1. Lock the regex to the front of the string
2. Ensure that there isn't any funny business at the end of the URL: the authenticated identity of a SciToken will have the form `<TOKEN ISSUER>,<TOKEN SUBJECT>` so we can use `,` as the URL terminator